events/ready.js

```diff
@@ -0,0 +1,6 @@
+const chainMessageAction = require('../eventActions/checkChainMessage');
```

Comment:
Not really sure where this is coming from. I don't see the required file in any of the committed files. Maybe a leftover from Horace?

Reply:
Yep, this is a leftover from copying over from Horace 😆 getting rid of it in next commit